CONTRIBUTING.md

```diff
@@ -58,33 +58,38 @@ Each new pull request should:
 
 ## Creating a new release
 
-### Minor/patch releases
-Minor and patch releases only need to change this repo. The Helm chart and deploy scripts
+### Patch releases
+Patch releases only need to change this repo. The Helm chart and deploy scripts
 will automatically pull in the latest changes.
 
-To deploy a minor or patch release, follow steps 2 and 3 from "Major releases" below.
+If the release involves changes to anything in the `deploy/` folder, it needs to be a minor
+or major release in order to prevent breaking the Helm chart.
 
-### Major releases
-Major releases need to change both this repository and the
+To deploy a patch release, follow steps 2 and 3 below.
+
+### Minor/Major releases
+Minor and major releases need to change both this repository and the
 [Helm chart repo](https://github.com/reactiveops/charts/).
 
 The steps are:
-1. Create a PR in the [charts repo](https://github.com/reactiveops/charts/)
-    1. Use a branch named `polaris-latest`
-    2. Bump the version number in:
+1. Modify the [Helm chart](https://github.com/reactiveops/charts/stable/polaris)
+    1. Bump the version number in:
         1. stable/polaris/README.md
         2. stable/polaris/Chart.yaml
         3. stable/polaris/values.yaml
-    3. **Don't merge yet!**
+    2. Make any additional changes necessary (e.g. new RBAC permissions)
 2. Create a PR for this repo
-    1. Bump the version number in:
+    1. Regenerate the deployment files. Assuming you've cloned the charts repo to `~/charts`:
+        1. `helm template ~/charts/stable/polaris/ --name polaris --namespace polaris --set templateOnly=true > deploy/dashboard.yaml`
+        2. `helm template ~/charts/stable/polaris/ --name polaris --namespace polaris --set templateOnly=true --set webhook.enable=true --set dashboard.enable=false > deploy/webhook.yaml`
```

Comment:
Since we're leveraging the chart for this, and the chart downloads polaris using a "latest minor" type version (0.2 at the time of writing this); This would introduce an unexpected behavior while the PR+Tag builds are running. Anyone who runs the `kubectl apply -f deploy.yml` before the images are pushed to quay, will accidentally download `${CURRENT_VERSION} - 1`. This could lead to very unexpected behavior for users while pipelines are in flight.

I think maybe a different approach could be to explicitly call out the specific version of code we're pushing as a part of the release. Using the same process, while the images are pushing to quay, users would get an `ImagePullBackOff` if they run the `kubectl apply -f deploy.yml` before the images are in quay.